k8s.gcr.io/images/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
should this be one level down in "k8s-staging-csi"?

Reply:
Oops, yes. Fixed.